cmake/kokkos_tpls.cmake

```diff
@@ -85,6 +85,17 @@ ENDIF()
 KOKKOS_IMPORT_TPL(ROCM INTERFACE)
 KOKKOS_IMPORT_TPL(LIBQUADMATH)
 
+IF (KOKKOS_ENABLE_IMPL_DESUL_ATOMICS)
+  IF(KOKKOS_ENABLE_DESUL_ATOMICS_FORCE_EXTERNAL)
```

Comment:
I am seeing
>$ git grep "Kokkos_ENABLE" cmake | wc -l
37
$ git grep "KOKKOS_ENABLE" cmake | wc -l
165

I think we only introduce the all-uppercase version for the `#cmakedefine`s. And otherwise it's safer to use the `Kokkos_*` version in `CMake` because that works even if the Kokkos option hasn't been called yet (since it's a `CACHE` variable).
Let me know if you want me to change. For this one, I mostly only went with what was already there.

Reply:
Yes change it